tap_googleads/client.py

```diff
@@ -106,18 +98,23 @@ def authenticator(self) -> OAuthAuthenticator:
             auth_headers=auth_headers,
         )
 
+    def _get_login_customer_id(self, context):
+        if self.login_customer_id:
+            return self.login_customer_id
```

Comment:
Following the behaviour you described, do you think it makes sense to _always_ use the `login_customer_id` from config if provided?

Reply:
Good question, the way I was thinking about it was that if a user defines login_customer_id in their config then it would override always and potentially now cause errors if access is denied. I'm not really sure all the use cases for this but I think the tap is now always choosing the correct login_customer_id so setting it yourself is unnecessary. The airbyte variant has this option still as well but its never accessed in the code.

Reply:
I am also unsure, but this should keep the same behaviour if a user is providing `login_customer_id` regardless. We can optimise later - maybe #32 has some historical context.